app/code/core/Mage/Core/sql/core_setup/upgrade-1.6.0.10-1.6.0.11.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Core
+ * @copyright  Copyright (c) 2024 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/** @var Mage_Core_Model_Resource_Setup $installer */
+$installer = $this;
+$installer->startSetup();
+
+$table = $installer->getTable('core/config_data');
+
+if ($installer->getConnection()->isTableExists($table)) {
+    $oldPath = 'admin/security/crate_admin_user_notification';
+    $newPath = 'admin/security/create_admin_user_notification';
+
+    $select = $installer->getConnection()->select()
+        ->from($table, ['config_id', 'path'])
+        ->where('path = ?', $oldPath);
+
+    $rows = $installer->getConnection()->fetchAll($select);
+
+    foreach ($rows as $row) {
+        $installer->getConnection()->update(
+            $table,
+            ['path' => $newPath],
+            ['config_id = ?' => $row['config_id']]
+        );
+    }
+}
```

Comment:
```suggestion
$installer->getConnection()->update(
    $installer->getTable('core/config_data'),
    ['path' => 'admin/security/create_admin_user_notification'],
    ['path = ?' => 'admin/security/crate_admin_user_notification']
);
```

it seems easier than selecting then cycling no?

I've tested out both and I can confirm they work